### PR TITLE
ASUB-8476 Redeem ota link

### PR DESCRIPTION
**Pull request by LauraPinilla**
**Be sure to run `npm test`, `npm run lint`, and write detailed test steps before requesting reviewers**

## Description
Logic and view for redeeming an ota nonce.

#### Jira Ticket: [THEMES-](https://arcpublishing.atlassian.net/browse/THEMES-)
https://arcpublishing.atlassian.net/browse/ASUB-8476

_Information about what you changed for this PR. Include any dependencies, companion PRs, relevant screenshots, or config changes._

## Test Steps

_Add detailed test steps a reviewer must complete to test this PR_

1. Checkout this branch `git checkout {branch name}`
2. Run fusion repo with linked blocks `npx fusion start -f -l {blocks to link}`
3. {Next ....}

### CODE REVIEW

**Review by accbjt — DISMISSED** [review]
LGTM

File: blocks/identity-block/intl.json [hunk 1]
```diff
@@ -1898,5 +1898,17 @@
...
+		"en": "Something went wrong. Try again."
+	},
+	"identity-block.ota-ivalid-login-link": {
+		"en": "Invalid login link"
+	},
+	"identity-block.ota-ivalid-login-link-subheadline": {
+		"en": "Looks like one time login link is invalid or expired. Please request a new log in link and try again."
+	},
+	"identity-block.ota-get-new-link": {
+		"en": "Get a new link"
```
Comment (malavikakoppula) [inline blocks/identity-block/intl.json]: @LauraPinilla Can you add these translation scripts to all the languages we are supporting?

**Review by malavikakoppula — CHANGES_REQUESTED** [review]

File: blocks/identity-block/intl.json (hunk 1 above)
Comment (LauraPinilla) [reply blocks/identity-block/intl.json]: Hi @malavikakoppula I just created the order in Lokalize. If you have a chance, do you mind taking a look at the PR excluding the langs, and once the order is completed I'll update the files with the new translations.

Comment (LauraPinilla) [reply blocks/identity-block/intl.json]: @malavikakoppula The lokalize order is now completed, and I have included the new keys for all languages we support. It's ready for a second look.
Thanks

**Review by malavikakoppula — APPROVED** [review]
LGTM

**Comment by vgalatro** [comment]
@LauraPinilla , this is all good to merge in.

**Pull request merged**